src/Components/BookmarkPost/BookmarkPost.jsx

```diff
@@ -0,0 +1,113 @@
+import { useThemeContext } from "Context";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import styles from "./BookmarkPost.module.css";
+
+export const BookmarkPost = () => {
+const dispatch = useDispatch();
+const allBookmarks = useSelector((state) => state.posts.bookmarks);
+const currentUser = useSelector((state) => state.auth.currentUser);
+const allUsers = useSelector((state) => state.users.users);
+const { setToggle } = useThemeContext();
+console.log(
+  allBookmarks
+    .map((post) => post.username)
+    .includes(allUsers.map((user) => user.username))
+);
+const something = allBookmarks.map((post) => post.username);
+console.log(
+  allUsers.filter((user) =>
+    user.username.includes(allBookmarks.map((post) => post.username))
+  )
+);
```

Comment:
Suggestions: Take care of the consoles and remove them all

Reply:
Thanks for pointing it out